draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2257,7 +2477,7 @@ trunk port bit rate which will also determine the Baud-rate, the
 modulation format, the FEC etc.
 
 The controller, when asked to set up a client connectivity service,
-needs to find a WDM tunnel suitable to comply the DWDM port
+needs to find a DWDM tunnel suitable to comply the DWDM port
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
needs to find a WDM tunnel suitable to comply the DWDM port
```